x/evm/types/params_legacy.go

```diff
@@ -0,0 +1 @@
+package types
```

Comment:
Is this file needed?

Reply:
Yes it is needed for migration

Reply:
Can you add a comment to the file with an explanation for this? Thanks.